examples/http-request-latency-burnrate.jsonnet

```
@@ -0,0 +1,18 @@
+local latency = import '../slo-libsonnet/latency-burn.libsonnet';
+
+{
+  local query = latency.burn({
+    metric: 'http_request_duration_seconds_bucket',
+    selectors: ['job="telemeter-server"','handler="upload"'],
+    # How much responce delay is too much.
+    latencyTheshold: 1,
```

```suggestion
    latencyThreshold: 1,
```
and I might even consider naming it latencyTarget. :) (Might need to rename a few wordings elsewhere)

I'm thinking of keeping it a bit more simple here. The problem is that now we take a int but the `le` label is always a string in Prometheus. This might become quite tricky if some floating points are being rounded in jsonnet for something like that.
Essentially: Let's have the user define whatever the le="1.5" is. So latencyTarget=le as for labels.

---

updated